app/models/user_programme_enrolment.rb

```diff
@@ -33,16 +33,14 @@ def assign_pathway(pathway_slug)
   end
 
   def assign_recommended_pathway(questionnaire_response)
-    recommender = Programmes::CSAccelerator::PathwayRecommender.new(questionnaire_response: questionnaire_response)
+    recommender = Programmes::CSAccelerator::PathwayRecommender.new(questionnaire_response:)
```

Comment:
Is `questionnaire_response:` implicitly being passed the arg of the same name?

Reply:
yeh this is a Ruby 3 thing, a lot tidier I think: https://blog.saeloun.com/2021/09/28/ruby-allow-value-omission-in-hash-literals.html